README.md

```diff
@@ -4,51 +4,62 @@
+
+# What people have to say ❤️
 > Very handy with those variants of the regular macros. Just add a `*` and you are inline def-ing the args! - Peter Strömberg (co-author of Calva)
 
 # Talk
-I gave a [talk](https://www.youtube.com/watch?v=WqilQulsJQc) about snitch at clojure Asia you can watch it 
+I gave a [talk](https://www.youtube.com/watch?v=WqilQulsJQc) about snitch at clojure Asia you can watch it
 here https://www.youtube.com/watch?v=WqilQulsJQc.
 
 # Usage
 ## Installation
-I recommend adding snitch to your `~/.lein/profiles.clj`.
+I recommend adding snitch to your `~/.lein/profiles.clj` or `deps.edn`.
 An example file would be
 ```clojure
-; profiles.clj
+; Leiningen profiles.clj
 {:user {:dependencies [[org.clojars.abhinav/snitch "0.1.16"]]}}
 
 {:dev {:dependencies [[org.clojars.abhinav/snitch "0.1.16"]]}}
 ```
-If you add it to your project's dev dependencies, you could add this line to your dev/user.clj
-```
+
+Once your repl is running you can evaluate:
+
+```clojure
 (require '[snitch.core :refer [defn* defmethod* *fn *let]])
 ```
-requiring these macros once, will intern these macros inside clojure.core & cljs.core, so you don't have to import them in every namespace.
+
+in the Clojure REPL and the macros will be interned inside clojure.core & cljs.core, so you don't have to import them in every namespace. It will also make the macros available for your ClojureScript namespaces if your ClojureScript REPL is cloned/spawned off of the Clojure REPL.
 
 ## Overview
 There are four macros `defn*`, `defmethod*` `*fn`, `*let`.
-They are drop-in replacements for `defn`, `defmethod`, `fn`, and `let`.
+They are drop-in replacements for `defn`, `defmethod`, `fn`, and `let`, respectively.
 
-These macros creates inline defs of the parameters passed to the functions,
+These macros create inline defs of the parameters passed to the functions,
 and also inside the let bindings of the functions.
-This makes it very "ergonomic" for repl-driven development.
+This makes it very ergonomic for repl-driven development.
+Calling functions defined using the `defn*`, and `*fn` macros will also define symbols that:
+1. give access to the return value of the functions. `(defn* foo [x y z] ...)` will define `foo<` which holds the return value of calling `foo`.
+2. reconstructs the function call. Defining `(defn* foo [x y z] ...)` and then calling `(foo 1 2 3)` will let you reconstruct the call by evaluating `foo>`, you can then evaluate this reconstructed list to repeat the function call. This may not seem like a big deal, but then remember that `foo` may be called indirectly from a chain of function calls, so being able to update the `foo` function and re-call it with the last arguments it was called with is quite convenient.
```

Comment:
These are so much clearer than what I came up with! thanks.

Reply:
You explained it very well in your Clojure Asia presentation. This is basically just that. 😄